### ISSUE TITLE
Updating to 87.0.4280.88

### ISSUE THREAD

**Issue by berkley4** [ungoogled-software/ungoogled-chromium-debian#186]
I've managed to update to 87.0.4280.88 and was wondering about making a pull request. Since it's not based on an official debian release, I'm unsure about making it to the 'debian_sid' destination branch and thereby losing sync with future debian git history.

In my own repo I've got things in a branch called '87.0.4280.88'. Please advise.

**Comment by Eloston** [comment]
Does Debian have any concrete plans to update their Chromium package?

**Comment by berkley4** [comment]
> Does Debian have any concrete plans to update their Chromium package?

Nothing I'd describe as concrete. There is a discussion on the debian bugtracker about forming a group maintenance team, plus one person working on an update. The wheels seem to turn very slowly, but I would expect it's more a question of when rather than if an update will materialise.

If you're reluctant about servicing two branches of UC, then I'd suggest making an unofficial-debian branch and not bothering with distributing binaries for it. You'd at least have an iron in the fire, so to speak.

**Comment by Eloston** [comment]
In that case, I'm reluctant to merge updates that would diverge us further from upstream. Instead, I could pin a new issue that refers to your fork until upstream manages to release a new update. Does this sound viable?

**Comment by berkley4** [comment]
Sorry for the delay, that sounds fine.

There has been some discussion and collaboration over at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=973848, and a debian maintainer has even submitted a package for review - https://mentors.debian.net/package/chromium/.

Hopefully we'll see debian releases resume in the near future, as it's a hard job to update from something that was so out of date.

**Comment by tpo** [comment]
Debian has [updated chromium to v87 in sid](https://packages.debian.org/sid/chromium). So now theoretically it should be possible for ungoogled-chromium to catch up?

**Comment by ghost** [comment]
The repository for the last 3 chromium uploads is at https://salsa.debian.org/mimi89999/chromium/

Discussion seems to be scattered across at least:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=972134
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=973848
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=974678

If Michel Le Bihan's work (linked above) is what's currently being uploaded, perhaps it could be used as a point of reference and possibly contribution?
It seems like they're rather open to any help they can get.

**Comment by Eloston** [comment]
Since upstream has pushed some updates, I think it's time to consider resuming development on our side.

Who wants to take ownership of updating?

* @berkley4 Since you've already made your own branch for v87, you probably have enough context to made an "official" change (i.e. making an update based on upstream code). Would you like to submit an update?
* More importantly, who is interested in maintaining this repo? @berkley4 or @braewoods, are either of you interested?

Regarding the unportable variant, I will keep on maintaining it for now. I don't know upstream's situation, but I don't think it's a good idea to depend exclusively on upstream until we are confident they can keep maintaining the package. I think one of unportable's benefits is its low maintenance cost (because it relies heavily on Portable Linux). There are some other benefits to unportable as well, but let's reconsider once we see how upstream's situation develops.

**Comment by SugaryHull** [comment]
@Eloston 
Debian has since pushed a non-maintainer update for chromium that pushes the version to [88.0.4324.96-0.1](https://packages.debian.org/source/sid/chromium). This was to address [26 outstanding CVEs](https://metadata.ftp-master.debian.org/changelogs//main/c/chromium/chromium_88.0.4324.96-0.1_changelog), and it appears to be working its way to Buster-Security as we speak.
~~Correction: [its migration is currently being blocked](https://qa.debian.org/excuses.php?package=chromium) by [issue #972134](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=972134)~~ Apologies for the error, mimi

**Comment by ghost** [comment]
@Eloston
Given the news given to us by @SugaryHull, I'll see about getting us updated to the new release in the debian upstream repository once we have chromium 88 available in our main repository. Unless you had other plans?

**Comment by SugaryHull** [comment]
There's currently a pull-request in the main ungoogled-chromium repo for an update to [88.0.4324.104](https://github.com/Eloston/ungoogled-chromium/pull/1342), and I am currently documenting issues with applying it to the current Debian and Portable Linux SKUs as they currently are. You can find my comments in the PR discussion.

**Comment by mimi89999** [comment]
Hello,
I made the recent NMU uploads of Chromium in Debian.

> Correction: its migration is currently being blocked by issue #972134

No, that issue is blocking Chromium from entering Bullseye. There is another issue with building Chromium for Buster that is being worked on.

Do _ungoogled-chromium_ patches for `88.0.4324.96` apply cleanly after the Debian patches?

**Comment by ghost** [comment]
@mimi89999 That's not how the ungoogled-chromium patches are applied. They are applied **before** the Debian patches. So they will apply cleanly once the upstream repository release is made.

**Comment by SugaryHull** [comment]
> I'll see about getting us updated to the new release in the debian upstream repository once we have chromium 88 available in our main repository

Would that be carried out by the OBS repos or through a community-contributed binary?

**Comment by ghost** [comment]
@SugaryHull OBS but I have to get it building locally first.

**Comment by Eloston** [comment]
@braewoods Sorry for the late response, feel free to go ahead with updating the branches. I can keep maintaining the unportable variant for the time being.

**Comment by ghost** [comment]
Chromium 88 is being worked on as we speak. This is no longer relevant.
